rmgpy/cantherm/commonTest.py

```diff
@@ -265,7 +266,20 @@ def testTransitionStateStatmech(self):
         job.includeHinderedRotors = self.useHinderedRotors
         job.applyBondEnergyCorrections = self.useBondCorrections
         job.load()
-
-
+        
+class testStatmech(unittest.TestCase):
+    """
+    Contains unit tests of statmech.py
+    """
+    def setUp(self):
+        cantherm = CanTherm()
+        jobList = cantherm.loadInputFile(os.path.join(os.path.dirname(os.path.abspath(__file__)),r'files/Benzyl/input.py'))
```

Comment:
change `files/Benzyl/input.py` to `test_data/Benzyl/input.py`, just a name convention for test data directory